### PR TITLE
Feature/crud with spa

### PR DESCRIPTION
**Pull request by ilyavveselov**

### CODE REVIEW

File: app/javascript/channels/servers_channel.js [hunk 1]
```diff
@@ -11,14 +11,17 @@ consumer.subscriptions.create("ServersChannel", {
 
   received(data) {
     var event;
-    console.log(data)
+    console.log("data: ", data);
```
Comment (owlscatcher) [inline app/javascript/channels/servers_channel.js]: [no-console](https://eslint.org/docs/latest/rules/no-console)

File: app/javascript/controllers/home_controller.js [hunk 2]
```diff
@@ -12,27 +12,34 @@ export default class extends Controller {
     });
 
     window.addEventListener('createServer', (event) => {
+      console.log("createServer: ", event.detail);
```
Comment (owlscatcher) [inline app/javascript/controllers/home_controller.js]: [no-console](https://eslint.org/docs/latest/rules/no-console)

**Review by owlscatcher — COMMENTED** [review]
Fix the notes and fill it up

File: app/javascript/controllers/home_controller.js (hunk 2 above)
Comment (ilyavveselov) [reply app/javascript/controllers/home_controller.js]: [5ca0148](https://github.com/digital-armstrong/Armstrong.WebServer/pull/8/commits/5ca0148d8b4fbf20e89b889c4b20795a53ce4299)

File: app/javascript/channels/servers_channel.js (hunk 1 above)
Comment (ilyavveselov) [reply app/javascript/channels/servers_channel.js]: [5ca0148](https://github.com/digital-armstrong/Armstrong.WebServer/pull/8/commits/5ca0148d8b4fbf20e89b889c4b20795a53ce4299)

**Review by owlscatcher — APPROVED** [review]

**Pull request merged**